src/data/README.md

```diff
@@ -18,7 +18,235 @@ TODO - describe the properties used in DevDot
+
+Example: `h2` with `400` size:
+
+```json5
+{
+  "type": "heading",
+  "heading": "Explore Waypoint Documentation",
+  "level": 2,
+  "size": 400,
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158818617-2b8ce029-ad41-4081-8701-869d51abf40b.png)
+
+</details>
+
+<!-- block type: get_started -->
+
+<details>
+<summary>Block type: <code>"get_started"</code></summary>
+
+The `"get_started"` block renders a heading, descriptive text, and a single link alongside a product icon.
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"get_started"` | Block type |
+| `product` | `"boundary" \| "consul" \| "nomad" \| "packer" \| "terraform" \| "vault" \| "vagrant" \| "waypoint" \| "sentinel" \| "hcp"` | Product icon to be shown. |
+| `heading` | `string` | Text for the heading |
+| `text` | `string` | Descriptive text shown below the heading |
+| `link` | `{ text: string, url: string }` | Link shown below the body text |
+
+Example: Waypoint `"get_started"` block
+
+```json5
+{
+  "type": "get_started",
+  "product": "waypoint",
+  "heading": "Introduction to Waypoint",
+  "text": "Welcome to Waypoint! This introduction section covers what Waypoint is, the problem Waypoint aims to solve, and how Waypoint compares to other software.",
+  "link": {
+    "text": "Get Started",
+    "url": "/waypoint/docs/intro"
+  }
+}
+```
+
+![](https://user-images.githubusercontent.com/4624598/158821262-03798dca-12e6-487b-ac3e-e8bab51be8b1.png)
+
+</details>
+
+<!-- block type: cards -->
+
+<details>
+<summary>Block type: <code>"cards"</code></summary>
+
+The `"cards"` block displays a grid of cards, each linked using a single `url`.
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `type` | `"cards"` | Block type |
+| `columns` | `2 \| 3` | The maximum number of columns |
+| `cards` | `Array<{ icon, iconBrandColor, heading, text, url, tags }>` | An array of `card` objects, described in detail below |
+
+Each item in the `cards` array has the following structure:
+
+| Property | Type | Details |
+| --- | --- | --- |
+| `icon` | (optional) `"IconBox" \| "IconConsulColor" \| "IconDocs" \| "IconDownload" \| "IconPackerColor" \| "IconProvider" \| "IconServer" \| "IconTerminal" \| "IconTerraformColor" \| "IconVaultColor"` | Optional icon to show at the top of the card. |
+| `iconBrandColor` | (optional) `"boundary" \| "consul" \| "nomad" \| "packer" \| "terraform" \| "vault" \| "vagrant" \| "waypoint" \| "neutral" \| "neutral-dark"` | Optional brand color override to apply to the icon. Defaults to the current product context. |
+| `heading` | `string` | Text for the card heading |
+| `text` | `string` | Text for the card body |
+| `url` | `string` | URL to link to |
+| `tags` | (optional) `Array<"boundary" \| "consul" \| "nomad" \| "packer" \| "terraform" \| "vagrant" \| "vault" \| "video" \| "waypoint">` | Optional array of tags, to be displayed as icons at the bottom of the card |
```

Comment:
💭 What do you think about linking to the TypeScript where `icon`, `iconBrandColor`, and `tags` are defined instead of listing them out? I worry we'll forget to update these docs if we add or remove options for them. 😅

Reply:
Ooh yeah I like that - will update! 👍

Reply:
✅  updated!